### PR TITLE
INT-579 Call Forwarding

### PR DESCRIPTION
**Pull request by nurutdinov**

### CODE REVIEW

File: talk/talk-api.yaml [hunk 1]
```diff
@@ -3276,3 +3273,23 @@ components:
         onCall:
           type: integer
           format: int64
+
+    BusinessCalendarInterval:
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:3277]: `BusinessHour`?

File: talk/talk-api.yaml [hunk 2]
```diff
@@ -2337,6 +2328,16 @@ components:
           $ref: '#/components/schemas/Id'
         whatsappEnabled:
           type: boolean
+        forwardingNumber:
```
[the remark this answers is not in the document]
Comment (BullDog57Rus) [reply talk/talk-api.yaml:2331]: and there will be required properties
number
twentyFourSeven
businessHours

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

File: talk/talk-api.yaml [hunk 3]
```diff
@@ -3276,3 +3257,41 @@ components:
...
+          type: string
+          format: x-time
+        closeHour:
+          type: string
+          format: x-time
+        weekday:
+          $ref: '#/components/schemas/DayOfWeek'
+        enabled:
+          type: boolean
+          description: If the business hour turned on or not
```
Comment (aehrenthal) [inline talk/talk-api.yaml:3279]:
```suggestion
          description: If the business hour is turned on or not.
```

**Review by aehrenthal — APPROVED** [review]

File: talk/talk-api.yaml [hunk 4]
```diff
@@ -2117,10 +2117,10 @@ components:
         twentyFourSeven:
           type: boolean
           description: If the auto receptionist works as business hours 24/7
-        businessHours:
+        businessHour:
```
Comment (BullDog57Rus) [inline talk/talk-api.yaml:2120]: `businessHours` here, as it is a list

**Review by BullDog57Rus — CHANGES_REQUESTED** [review]

**Review by BullDog57Rus — APPROVED** [review]

**Pull request merged**